scaleway/config.go

```diff
@@ -114,7 +114,7 @@ func createRetryableHTTPClient(shouldLog bool) *client {
 	c.RetryMax = 3
 	c.RetryWaitMax = 2 * time.Minute
 	c.Logger = log.New(os.Stderr, "", 0)
-	c.RetryWaitMin = time.Minute
+	c.RetryWaitMin = time.Second * 2
```

Comment:
You don't want to revert it?

Reply:
Let's keep the min low and increase it if we need (max remains 2 minutes)